### ISSUE TITLE
jNativehook.dll is used by another process

### ISSUE THREAD

**Issue by wizzyclems** [kwhat/jnativehook#75]
Hi Kwhat, 

Thanks for a great api. 

I'm creating two apps that must run on the same system and listen on Global Key events. Both are using jnativehook. Thing is, when I run one I can't run the other as the jnativehook.dll has been locked by the first.

How can you help with this.

Cheers.

**Comment by kwhat** [comment]
This might be fixed in 2.0.3 that I haven't released yet.  Try this nightly and see if it is still an issue: https://oss.sonatype.org/content/repositories/snapshots/com/1stleg/jnativehook/2.0.SNAPSHOT/jnativehook-2.0.20151028.044132-23.jar

**Comment by wizzyclems** [comment]
Hi Kwha, after some research, I discovered that the issue is the fact that u dropped the jnativehook dll in the java.io.tmpdir; which means that all two app where trying to access the same file ergo the error. 
I had to change that and repackage the api and it worked. Thanks much for your helpful api. Cheers

**Comment by kwhat** [comment]
Yes, that is what the jar addresses.  I stopped trying to overwrite the dll in java.io.tmpdir if it already exists in 2.0.3.  I am waiting on some post event issues on Linux that should be fixed today before releasing earlier today.

**Comment by kwhat** [comment]
Alright, this should be fixed by https://github.com/kwhat/jnativehook/releases/tag/2.0.3 

duplicate of issue #69
